contracts/MiniMeToken.sol

```diff
@@ -541,6 +541,7 @@ contract MiniMeToken is Controlled {
 ///  In solidity this is the way to create a contract from a contract of the
 ///  same class
 contract MiniMeTokenFactory {
+    event NewCloneToken(address indexed _cloneToken, uint _snapshotBlock);
```

Comment:
Maybe we could use a different signature here, to avoid potential confusion:

`NewFactoryCloneToken(address indexed _cloneToken, address indexed _parentToken, uint256 _snapshotBlock)`